spring-vault-core/src/main/java/org/springframework/vault/exceptions/package-info.java

```diff
@@ -0,0 +1,7 @@
+/**
+ * Spring Vault Client Exceptions. Some Exceptions still remain in other packages
+ * for compatibility purposes.
+ */
+@org.springframework.lang.NonNullApi
+@org.springframework.lang.NonNullFields
+package org.springframework.vault.exceptions;
```

Comment:
We should put exceptions into the packages where they are exported/primarily thrown. An exception package is always a dependency magnet that does not properly reflect the structure of the code.